lib/matplotlib/rcsetup.py

```diff
@@ -733,9 +733,9 @@ def cycler(*args, **kwargs):
     Creates a :class:`cycler.Cycler` object much like :func:`cycler.cycler`,
     but includes input validation.
 
-    cyl(arg)
-    cyl(label, itr)
-    cyl(label1=itr1[, label2=itr2[, ...]])
+    cyler(arg)
+    cyler(label, itr)
+    cyler(label1=itr1[, label2=itr2[, ...]])
```

Comment:
still has a typo. `cyl` was the original name, but then we decided it was ok to collide with the `cycler` from the `cycler` package.